jerry-core/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ message(STATUS "MEM_HEAP_SIZE_KB          " ${MEM_HEAP_SIZE_KB})
 set(INCLUDE_CORE
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${CMAKE_CURRENT_SOURCE_DIR}/api"
-    "${CMAKE_CURRENT_SOURCE_DIR}/debugger"
+    "${CMAKE_CURRENT_SOURCE_DIR}/dbg"
```

Comment:
(no question: if I ever had the chance, I'd get rid of them from `jcontext` and `jmem`, too. as explained at #1795. fortunately, it's not up to me, only :) )